components/edit-collective/tiers/EditTierModal.js

```diff
@@ -125,6 +141,18 @@ function FormFields({ collective, types, values }) {
           )}
         </StyledInputFormikField>
       )}
+      {(![FUND].includes(collective.type) || types?.length === 1) &&
```

Comment:
Why not for funds?

Reply:
To be honest, I dont know the specific rule here. This is replicating the logic used on the previous tier page, the tier type option is only rendered when `![FUND].includes(collective.type) || types?.length === 1`. And the taxes info are only rendered when the tier type option is rendered.

I can refactor to render this inside the above component to make the dependency clear, if we keeping the existing behavior.